### DISCUSSION TITLE
You need name attribute in radio buttons when you set it .defer

### DISCUSSION THREAD

**Discussion by nshiro** [livewire/livewire#4545] [7. Bugs (archived)] [answered]
@joshhanley
I saw your new video 'Radio Buttons'.
https://laravel-livewire.com/screencasts/form-radios

And I accidentally noticed when you add `.defer`, you also need `name` attribute.
Otherwise you can select all radio buttons before you send a form.

![2022-01-19_11h22_16](https://user-images.githubusercontent.com/14008307/150051656-2c117abf-6319-4a78-9351-ba41b226b86d.png)

![2022-01-19_11h13_51](https://user-images.githubusercontent.com/14008307/150051673-bbe7c653-1feb-4fea-aa87-9d9537dba6f9.png)

I just wanted to let you know. 
Thank you for your hard work.

**Comment by joshhanley** [comment, marked as answer]
@nshiro thanks!

Yeah that was an error, as browsers need the name attribute on radio buttons so it knows when to group it together.

It works fine without defer as Livewire's re-rendering fixes it, but doesn't work when using defer as the browser is then controlling which should be selected/deselected.
